composer.json

```diff
@@ -2,6 +2,7 @@
   "name": "swedbank-pay/swedbank-pay-sdk-php",
   "type": "library",
   "description": "The Swedbank Pay SDK for PHP simplifies integrations against Swedbank Pay's API Platform by providing native PHP interface towards the REST API.",
+  "version": "4.0.0",
```

Comment:
`version` is dynamically added by the build. Don't add it statically here, please.

Reply:
@aait Packagist / composer doesn't add `version`. It created problem with version detection.